sport/test/football/feed/CompetitionsTest.scala

```diff
@@ -0,0 +1,207 @@
+package football.feed
+
+import java.time.{Clock, Duration, LocalTime, ZoneId, ZonedDateTime}
+import java.time.format.DateTimeFormatter
+import org.scalatest._
+import feed.Competitions
+import football.model.CompetitionTestData
+import model.Competition
+import pa.{
+  Fixture,
+  FootballMatch,
+  LeagueStats,
+  LeagueTableEntry,
+  LeagueTeam,
+  MatchDay,
+  MatchDayTeam,
+  Result,
+  Round,
+  Stage,
+}
+
+class CompetitionsTest extends FreeSpec with Matchers with OptionValues {
+  "Competitions" - {
+    "isAMatchInProgress return true if there's a match that started 1 minute ago" in {
+      val matches: Seq[FootballMatch] =
+        FootballTestData.matchesWithLiveMatchAtCurrentMinusDuration(Duration.ofMinutes(1))
+      val testCompetition = FootballTestData.competitions(1).copy(matches = matches)
+      val competitions = Competitions(Seq(testCompetition))
+
+      val isAMatchInProgress = competitions.isAMatchInProgress(competitions.matches, FootballTestData.clock)
+
+      isAMatchInProgress should equal(true)
+    }
+
+    "isAMatchInProgress return true if there's a match that started 5 minute ago" in {
+      val matches: Seq[FootballMatch] =
+        FootballTestData.matchesWithLiveMatchAtCurrentMinusDuration(Duration.ofMinutes(5))
+      val testCompetition = FootballTestData.competitions(1).copy(matches = matches)
+
+      val test = FootballTestData.today
+      val competitions = Competitions(Seq(testCompetition))
+
+      val isAMatchInProgress = competitions.isAMatchInProgress(competitions.matches, FootballTestData.clock)
+
+      isAMatchInProgress should equal(true)
+    }
+  }
+
+  // I had trouble using the original FootballTestData because it needed testFootballClient
+  // And in my test I didn't need any client at all. As it's testing the Competitions methods
+  // For now I copied the things from the original FootballTestData until I figure out how to fix this
```

Comment:
@adamnfish we may need some help here. We already had `FootballTestData` but I didn't manage to use it for this `CompetitionsTest` mostly because it has `testFootballClient` that needs to be implemented. But in this test we don't need to use any client. because it's just testing the methods within the Competitions trait. 

For now I copied the parts of the content in `FootballTestData` to here just to be able to progress with the tests. But I know we need to fix this along the way.

Reply:
This was fixed by using only the FootballTestData rather than extending the trait